st2actions/st2actions/utils/param_utils.py

```diff
@@ -215,15 +215,25 @@ def _do_render_params(renderable_params, context):
     env = Environment(undefined=StrictUndefined)
     rendered_params = {}
     rendered_params.update(context)
+
+    keys_to_delete = []
     while len(renderable_params) != 0:
         for k, v in six.iteritems(renderable_params):
             template = env.from_string(v)
+
             try:
                 rendered = template.render(rendered_params)
                 rendered_params[k] = rendered
-            except:
+                keys_to_delete.append(k)
+            except Exception:
+                # TODO: Maybe we should throw here and abort action execution?
```

Comment:
I actually believe throwing is a right course of action - we shouldn't just skip and ignore a problem if rendering a parameter fails.

@manasdk et al. wdyt?